tests/test_main.py

```diff
@@ -704,7 +705,7 @@ def test_systemdetails_hidden_auth():
 
 # -- generic tests for all Charmcraft commands
 
-all_commands = list.__add__(*[cgroup.commands for cgroup in COMMAND_GROUPS])
+all_commands = list(itertools.chain(*(cgroup.commands for cgroup in COMMAND_GROUPS)))
```

Comment:
Did this come from a linter issue? If not, please remove this change here (though a separate PR with this fix would be much appreciated!)

Reply:
It is a invalid code when `COMMAND_GROUPS` has more than 2 items.
`TypeError: expected 1 argument, got 2`

Reply:
Ahh, I see. All good then, thanks!